webrtc.html

```diff
@@ -524,26 +528,31 @@
 
           <li>
             <p>If the length of the <code>operations</code> array is exactly 1,
-            execute the function from the front of the queue
-            asynchronously.</p>
+            execute the function from the front of the queue.</p>
+          </li>
+
+          <li>
+            <p>When the promise returned by the function is settled,
+            remove the corresponding object from the
+            <code>operations</code> array. After removal, if the array
+            is non-empty, execute the first object queued.  After
+            execution, settle <code>p</code> with the result from the
+            promise returned by the function.</p>
```

Comment:
or maybe "Upon fulfillment or rejection of the promise returned by the function, settle *p* correspondingly."